### PR TITLE
readds 88mod stock, makes it buff 88mod quite a bit, reduces basefalloff

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
88mod is pretty bad, just about nobody ever uses it, and honestly the stock should make it into a different gun overall.
-It increases aim slowdown, and wield delay to rifle level
-Gives it +1 to burst and 0.15 fire delay

## Why It's Good For The Game
Makes the 88mod into a more unique firearm rather than a meme you'll see a PVT pick once a year.

## Changelog
:cl:
balance: 88mod has it's stock back, makes it buff the guns firerate, burst amount at the cost of slowdown and wield speed increases. Falloff reduced to 0.5 from 1.
/:cl:

### CODE REVIEW

**Comment by Alterist** [comment]
YES

**Comment by MereWalker** [comment]
Im not really a fan of the mod too much. I use the tp45 often than most other guns. I however look forward to this. Could be a good medic gun.

**Comment by davidchan13** [comment]
I'd you're trying to buff the automatic pistol I'd suggest giving it more capacity than it's semiauto counterpart per mag (21 on tp14 vs 18 on 88mod)

**Review by Kuro020 — APPROVED** [review]
👍

File: code/modules/projectiles/gun_attachables.dm [hunk 1]
```diff
@@ -906,6 +906,60 @@ inaccurate. Don't worry if force is ever negative, it won't runtime.
...
+	pixel_shift_y = 11
+	accuracy_mod = 0.15
+	aim_speed_mod = 0.2
+	recoil_mod = -3
+	scatter_mod = -20
+	delay_mod = -0.05 SECONDS // 0.2 to 0.15
+	burst_mod = 1
+	movement_acc_penalty_mod = 0.1
+	var/obj/item/storage/internal/pockets = /obj/item/storage/internal/pockets/vp70holster
+
```
Comment (TiviPlus) [inline code/modules/projectiles/gun_attachables.dm:932]: Autodoc

File: code/modules/projectiles/gun_attachables.dm [hunk 2]
```diff
@@ -906,6 +906,60 @@ inaccurate. Don't worry if force is ever negative, it won't runtime.
...
+/obj/item/attachable/stock/vp70/Initialize()
+	. = ..()
+	pockets = new pockets(src)
+
+/obj/item/attachable/stock/vp70/attack_hand(mob/living/user)
+	if(loc == user && length(pockets.contents))
+		var/obj/item/I = pockets.contents[length(pockets.contents)]
+		return I.attack_hand(user)
+	else if(pockets.handle_attack_hand(user))
+		return ..()
```
Comment (TiviPlus) [inline code/modules/projectiles/gun_attachables.dm:948]: Uneded elsifs

File: code/modules/projectiles/gun_attachables.dm [hunk 3]
```diff
@@ -906,6 +906,60 @@ inaccurate. Don't worry if force is ever negative, it won't runtime.
...
+
+/obj/item/attachable/stock/vp70/MouseDrop(obj/over_object)
+	if(pockets.handle_mousedrop(usr, over_object))
+		return ..(over_object)
```
Comment (TiviPlus) [inline code/modules/projectiles/gun_attachables.dm:952]: uneeded args passed to ..()

File: code/modules/projectiles/gun_attachables.dm [hunk 4]
```diff
@@ -906,6 +906,60 @@ inaccurate. Don't worry if force is ever negative, it won't runtime.
...
+
+/obj/item/attachable/stock/vp70/attackby(obj/item/I, mob/user, params)
+	. = ..()
```
Comment (TiviPlus) [inline code/modules/projectiles/gun_attachables.dm:955]: uneeded . = ..()
just use ..()

**Review by TiviPlus — CHANGES_REQUESTED** [review]

**Comment by Pariah919** [comment]
ehh
i'll fix this up one day, on further thought

**Pull request closed without merging**